src/common/config/decrypt.tsx

```diff
@@ -64,28 +64,40 @@ export const decryptWalletOrSigner = async (
   progressCallback: (progress: number) => void
 ): Promise<Wallet | ConnectedSigner> => {
   const provider = config.network === "local" ? new providers.JsonRpcProvider() : getDefaultProvider(config.network);
-
   if (isWalletOption(config.wallet)) {
     const decryptedWallet = await Wallet.fromEncryptedJson(config.wallet, password, progressCallback);
     const connectedWallet = await decryptedWallet.connect(provider);
     return connectedWallet;
   } else {
-    const kmsCredentials = {
-      accessKeyId: config.wallet.accessKeyId, // credentials for your IAM user with KMS access
-      secretAccessKey: password, // credentials for your IAM user with KMS access
-      region: config.wallet.region,
-      keyId: config.wallet.kmsKeyId,
-    };
+    switch (config.wallet.type) {
+      case "ENCRYPTED-JSON":
```

Comment:
can we have a sample config for this type?

Reply:
updated all the current config file that are using ENCRYPTED-JSON.